apps/portal/components/pagesComponents/_imagesScreen/components/multiItemsSelect/MultiItemsSelect.tsx

```
@@ -229,30 +241,41 @@ const MultiItemsSelect: React.FC<IMultiItemsSelectProps> = ({
       setLoader(false);
     }
 
     if (hasError) return;
 
-    infoMessage('Video deleted');
+    infoMessage(t('toasts.videoDeleted'));
   };
-
+  {
+    t('toasts.itemsDeleted');
+  }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)